src/main/java/com/cryptomorin/xseries/XBiome.java

```diff
@@ -240,9 +242,14 @@ public Biome getBiome() {
         // Apparently setBiome is thread-safe.
         return CompletableFuture.runAsync(() -> {
             for (int x = 0; x < 16; x++) {
-                for (int z = 0; z < 16; z++) {
-                    Block block = chunk.getBlock(x, 0, z);
-                    if (block.getBiome() != biome) block.setBiome(biome);
+                // y loop for 1.16+ support (vertical biomes).
+                // As of now increasing it by 4 seems to work.
+                // This should be the minimal size of the vertical biomes.
```

Comment:
> This should be the minimal size of the vertical biomes.

Can you also reference the source that why it should be specifically 4?

Reply:
It'd be even better if there's a source available from either Spigot or Minecraft that explains the biome system in 1.16

Reply:
> > This should be the minimal size of the vertical biomes.
> 
> Can you also reference the source that why it should be specifically 4?

Trial and error, 4 seemed to be the biggest number before issues started occuring :shrug:

Reply:
Ah ok, I see. Thanks